source/_docs/composer-drupal-8.md

```diff
@@ -9,6 +9,16 @@ contributors:
 
 [Composer](https://getcomposer.org/) is a dependency management tool that allows PHP code to be more easily shared across projects. As Drupal 8 was written, many of its subsystems written specifically for Drupal were replaced by more widely used packages. For instance, Drupal's notoriously unwieldy function for making external requests, `drupal_http_request()`, [was replaced](https://www.drupal.org/node/1862446) by [Guzzle](http://guzzlephp.org/). Guzzle is included in Drupal 8 via Composer.
 
+Before using Composer to manage your site development you should choose a workflow. By default, Pantheon controls and distributes core updates via git commits, which conflicts with updates processed by Composer. We suggest developers using Composer install their sites with the [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer), which moves Drupal to a "nested docroot". The repository README file covers the two common methods of deployment.
```

Comment:
Are we going to describe multiple workflows for the user to choose between? Certainly there is a choice between "Pantheon upstream" vs. "composer update"; perhaps in this document, the choice should be limited to those two options. The GitHub workflow can be introduced later.

`By default, Pantheon controls and distributes core updates via git commits, which conflicts with updates processed by Composer.` is a little unwieldy. Maybe something like "which can cause conflicts with files updated by Composer" would be clearer.

 `The repository README file covers the two common methods` - at this point it's not clear which two methods you are talking about. c.f. comment above.